src/transformers/models/deberta/modeling_deberta.py

```diff
@@ -539,6 +599,45 @@ def pos_dynamic_expand(pos_index, p2c_att, key_layer):
     return pos_index.expand(p2c_att.size()[:2] + (pos_index.size(-2), key_layer.size(-2)))
 
 
+@torch.jit.script
+def get_attention_mask_on_device(device_discriminator, using_ids: bool):
+    input_shape = device_discriminator.size() if using_ids else device_discriminator.size()[:-1]
+    return torch.ones(input_shape, device=device_discriminator.device)
+
+
+@torch.jit.script
+def get_token_type_ids_on_device(device_discriminator, using_ids: bool):
+    input_shape = device_discriminator.size() if using_ids else device_discriminator.size()[:-1]
+    return torch.zeros(input_shape, dtype=torch.long, device=device_discriminator.device)
+
+
+@torch.jit.script
+def scaled_size_sqrt(query_layer, scale_factor: int):
+    return torch.sqrt(torch.tensor(query_layer.size(-1), dtype=torch.float) * scale_factor)
```

Comment:
why does this have to be define as such? (FMI) known torch script issue or expected way to do so?